trailblazer/constants.py

```diff
@@ -53,12 +53,15 @@ def list(cls):
 
 
 TOWER_STATUS: Dict[str, str] = {
+    "NEW": TrailblazerStatus.PENDING.value,
     "SUBMITTED": TrailblazerStatus.PENDING.value,
     "RUNNING": TrailblazerStatus.RUNNING.value,
     "SUCCEEDED": TrailblazerStatus.COMPLETED.value,
-    "FAILED": TrailblazerStatus.FAILED.value,
-    "CANCELLED": TrailblazerStatus.CANCELLED.value,
     "COMPLETED": TrailblazerStatus.COMPLETED.value,
+    "CACHED": TrailblazerStatus.COMPLETED.value,
```

Comment:
Or is it still running?

Reply:
no, cached jobs are the ones that run successfully completed from a previous run (i.e. when resuming an analysis). So they are completed jobs